GSA-TTS/FAC
# FAC Logo Update

# FAC Logo Update - New SVG and PNG

Issue: #2189 

## Changes:
Previously, the logo in the top left was an SVG icon and some stylized text, which was very fancy and equivalent to the full FAC Logo. But, the CORS error in our cloud environments messes up that text, and it looks wrong. With that in mind:

1. The stylized text is replaced by an SVG version of the FAC full logo. No relying on the text to load properly. A PNG version is also included in the PR for potential future use. 
2. The accordions in the header (the drop downs) now have darker text (#171717). 

## How to test:
Switch to this branch and run normally. Verify that the log in the top left looks correct, and that the accordions have appropriately dark text. 

## Screenshots:
In this PR:
![image](https://github.com/GSA-TTS/FAC/assets/91098850/cfa152f1-ea93-4360-ba04-c06945fc2543)

On [app.fac.gov](https://app.fac.gov/audit/):
![image](https://github.com/GSA-TTS/FAC/assets/91098850/d4da089a-a9f7-4c01-846a-7f9c620cf1a1)

## PR checklist: submitters


## PR checklist: reviewers

        
The larger the PR, the stricter we should be about these points.
